### PR TITLE
Set default DISCOVERY_DISPLAY_ROWS

### PR DESCRIPTION
**Pull request by tshastry**
In my environment, the default value ended up as an empty string, which failed to coerce to an integer

### CODE REVIEW

**Review by mgoldey — APPROVED** [review]
Good catch

File: interactive_sdk.py [hunk 1]
```diff
@@ -20,7 +20,7 @@
 DISCOVERY_INTENTS = os.environ.get('DISCOVERY_INTENTS', '').split(',')
 
 # None here prints all datafram rows
-pd.set_option('display.max_rows', int(os.environ.get('DISCOVERY_DISPLAY_ROWS')))
+pd.set_option('display.max_rows', int(os.environ.get('DISCOVERY_DISPLAY_ROWS',0)))
```
Comment (cbrochtrup) [inline interactive_sdk.py:23]: This will undo the change if the environment variable is set (note the comment above). We need this to be `None` for no all dataframes to print by default.
```suggestion
display_rows = int(os.environ.get('DISCOVERY_DISPLAY_ROWS',0))
pd.set_option('display.max_rows', display_rows if display_rows > 0 else None)
```

**Review by cbrochtrup — CHANGES_REQUESTED** [review]
Nice catch on this broken code but we should set the default differently.

**Review by cbrochtrup — APPROVED** [review]

**Comment by tshastry** [comment]
@cbrochtrup can you make the PR into the private GK repo as well?
